pkg/controller/mobiledeveloperconsole/mobiledeveloperconsole.go

```diff
@@ -212,6 +214,22 @@ func newMDCDeploymentConfig(cr *mdcv1alpha1.MobileDeveloperConsole) (*openshifta
 									Name:  "OPENSHIFT_HOST",
 									Value: cfg.OpenShiftHost,
 								},
+								{
+									Name:  "IDM_DOCUMENTATION_URL",
+									Value: os.Getenv("IDM_DOCUMENTATION_URL"),
```

Comment:
@craicoverflow can you read the env vars similar to here: https://github.com/aerogear/mobile-developer-console-operator/blob/842a58a09d8d0cc950ab2dc4c02c11dcc39709d0/pkg/config/config.go#L27

It is better to keep these env vars in a single place.

Reply:
Sure yes, makes sense. I'm not sure why I didn't do this in the first place.